src/main/java/de/doubleslash/keeptime/view/ProjectsListViewController.java

```diff
@@ -56,19 +50,18 @@
    private final Controller controller;
    private final Stage mainStage;
    private final Map<Project, Label> elapsedProjectTimeLabelMap = new HashMap<>();
-   private final ListView<Project> availableProjectsListView;
    private final Map<Project, Node> projectSelectionNodeMap;
    private final FilteredList<Project> filteredData;
 
-   private boolean hideable;
+   private final boolean hideable;
+   private ManageProjectController manageProjectController;
```

Comment:
is this field needed? is it not sufficient to be available localy in the needed method?

Reply:
Done.